### PR TITLE
Log when dataset download uri not found

### PR DESCRIPTION
**Pull request by maukoquiroga**
Connected to #251 

capture d ecran 2018-03-19 a 12 02 21

### CODE REVIEW

File: priv/repo/datasets.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
 }, {
     "spatial": "Communauté d'agglomération Royan Atlantique",
     "coordinates": [-1.027697, 45.625611],
-    "slug": "donnees-gtfs-du-reseau-de-transport-public-carabus-1",
```
Comment (Tristramg) [inline priv/repo/datasets.json]: Why this commit? The old slug seems to be the correct one:

http://next.data.gouv.fr/fr/datasets/donnees-gtfs-du-reseau-de-transport-public-carabus-1/

Comment (Tristramg) [reply priv/repo/datasets.json]: Forget what I said. I was on next… :man_facepalming:

**Comment by maukoquiroga** [comment]
Try to rebase rather than merge, before merging.

**Comment by Tristramg** [comment]
Noted!

**Pull request merged**